backend/src/main/kotlin/org/loculus/backend/controller/FilesController.kt

```diff
@@ -23,8 +33,44 @@ class FilesController(
     private val filesDatabaseService: FilesDatabaseService,
     private val s3Service: S3Service,
     private val groupManagementPreconditionValidator: GroupManagementPreconditionValidator,
+    private val submissionDatabaseService: SubmissionDatabaseService,
+    private val accessionPreconditionValidator: AccessionPreconditionValidator,
 ) {
 
+    @GetMapping("/get/{accession}/{version}/{fileField}/{fileName}")
```

Comment:
I guess we need the `get` in here because otherwise an accession could clash with the `requestUpload`? It's not great to have the `get` in the path I think, but I can't think of another solution either.

Reply:
alright, I'd say let's just leave it the way it is!